src/components/SubHeader/subHeader.css

```diff
@@ -0,0 +1,99 @@
+
+/* sub header css start */
+.subHeaderText {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 50px 0;
+}
+
+.subHeader {
+  display: flex;
+  flex-direction: column;
+}
+
+.subHeaderTitle {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 60px 25px 10px;
+  font-weight: 600;
+}
+.headLine, .subHeadLine{
+  font-style: normal;
+  font-weight: 600;
+  font-size: 80px;
+  line-height: 88px;
+  text-align: center;
+  letter-spacing: 0.2px;
+  color: #FFFFFF;
+}
+
+.subHeadLine {
+  max-width: 60%;
+}
+
+.subHeaderContent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 25px;
+}
+
+.subHeaderContent .content{
+  font-style: normal;
```

Comment:
font style is not same as figma.

Reply:
Yes, its in figma]
![Screenshot (18)](https://user-images.githubusercontent.com/96838853/175780467-b280ab26-9ec2-4083-bfb6-9fbc359c3b48.png)